### PR TITLE
Addition of FORM.yml

### PR DESCRIPTION
**Pull request by AbhijithGanesh**
An attempt to fix Issue #982. I haven't personally added the template to any of my repositories because I didn't find the option to add a template. Can this be checked @brylie . Thank you

### CODE REVIEW

**Review by vishalda — CHANGES_REQUESTED** [review]
Hey, I guess it would be good if you change 'Issue Template' to 'Describe your Issue' and 'What happened' to 'Expected Behavior'. And also you can enable Issue tab from 'Settings>Options' for Issue templates to be displayed on your repository.

**Comment by AbhijithGanesh** [comment]
> Hey, I guess it would be good if you change 'Issue Template' to 'Describe your Issue' and 'What happened' to 'Expected Behavior'. And also you can enable Issue tab from 'Settings>Options' for Issue templates to be displayed on your repository.

Yes I shall make changes to the template , I wanted to just dry run this before I make active changes

**Comment by AbhijithGanesh** [comment]
Can this be listed under mentoring ?

File: .github/ISSUE_TEMPLATE/FORM.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Issue Template
+description: File/Create an Issue
```
Comment (brylie) [inline .github/ISSUE_TEMPLATE/FORM.yml]:
```suggestion
description: Create an Issue

```

File: .github/ISSUE_TEMPLATE/FORM.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+name: Issue Template
+description: File/Create an Issue
+title: "[Issue]: <Your Issue's brief Description goes here> "
```
Comment (brylie) [inline .github/ISSUE_TEMPLATE/FORM.yml]:
```suggestion
title: "<Enter a brief description here> "

```

File: .github/ISSUE_TEMPLATE/FORM.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+name: Issue Template
+description: File/Create an Issue
+title: "[Issue]: <Your Issue's brief Description goes here> "
+labels: [Bug, Triage, Idea,Other]
```
Comment (brylie) [inline .github/ISSUE_TEMPLATE/FORM.yml]:
```suggestion
labels: [Triage]

```

File: .github/ISSUE_TEMPLATE/FORM.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+assignees:
+  - Brylie
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this Bug/Issue report!
+  - type: input
+    attributes:
+      label: Issue Template
```
Comment (brylie) [inline .github/ISSUE_TEMPLATE/FORM.yml]:
```suggestion
      label: Description

```

File: .github/ISSUE_TEMPLATE/FORM.yml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+      required: true
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
```
Comment (brylie) [inline .github/ISSUE_TEMPLATE/FORM.yml]:
```suggestion
      label: What should have happened?
      description: If this issue is causing unexpected behavior, what should happen differently?
      placeholder: I expected...
    validations:
      required: false

```

File: .github/ISSUE_TEMPLATE/FORM.yml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+        - Firefox
+        - Chrome
+        - Safari
+        - Microsoft Edge
+  - type: textarea
+    id: multimedia
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      placeholder: Please upload your multimedia(screenshots and video) here
```
Comment (brylie) [inline .github/ISSUE_TEMPLATE/FORM.yml]:
```suggestion
      label: Further details
      description: Please copy and paste any relevant log output or multimedia.
      placeholder: Please add log file or your multimedia (screenshots and video) here

```

File: .github/ISSUE_TEMPLATE/FORM.yml [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
```
Comment (brylie) [inline .github/ISSUE_TEMPLATE/FORM.yml]:
```suggestion

```

**Review by brylie — APPROVED** [review]

**Comment by brylie** [comment]
@all-contributors please add @AbhijithGanesh for projectManagement

**Pull request merged**